src/Plaid/Entity/WebhookPayload.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Going.Plaid.Exceptions;
+using Newtonsoft.Json;
+
+namespace Going.Plaid.Entity
+{
+	/// <summary>
+	/// Represents the object Plaid will send to the Webhook
+	/// URL.
+	/// </summary>
+	public class WebhookPayload
+	{
+		/// <summary>
+		/// Represents which type of webhook is being delivered.
+		/// </summary>
+		[JsonProperty("webhook_type")]
+		public WebhookType WebhookType { get; set; }
+
+		/// <summary>
+		/// Represents additional information about the webhook payload
+		/// </summary>
+		[JsonProperty("webhook_code")]
+		public WebhookCode WebhookCode { get; set; }
+
+		/// <summary>
+		/// The <see cref="Item.Id"/> of the <see cref="Item"/> associated 
+		/// with this webhook, warning, or error
+		/// </summary>
+		[JsonProperty("item_id")]
+		public string ItemId { get; set; } = null!;
```

Comment:
It would be helpful to add a <remark> that this value is not set for Assets (https://plaid.com/docs/#assets-webhooks) and Payment Initiation (https://plaid.com/docs/#payment-initiation-webhooks) webhooks in Plaid API v2019-05-29.

Reply:
I'm silently intentionally ignoring Assets and Payment Initiation because the APIs for those do not exist yet in this library. I'll edit Type and Payload for both if/when we come back. This should be `string?` though, I'll fix that.